buildSystem/dependencies.gradle

```diff
@@ -26,6 +26,12 @@ ext {
     kointest = "2.1.5"
     androidxespressocore = "3.2.0"
     androidxjunit = "1.1.1"
+    roomruntime = "2.2.5"
+    roomktx = "2.2.5"
+    roomtest = "2.2.5"
```

Comment:
its neccesary that every room dependecy has its how version?

Reply:
grouped the dependencies with the same version together, thanks